src/commands/dev/index.js

```diff
@@ -136,7 +114,7 @@ const startProxyServer = async ({ flags, settings, site, log, exit, addonUrls })
       exit(1)
     }
   } else {
-    url = await startProxy(settings, addonUrls, site.configPath, site.root)
```

Comment:
`addonUrls` had all addons Urls (not a single addon Urls) so renamed it to `addonsUrls`.

Those are use to redirect local requests to `http://localhost:8888/.netlify/<addon>/*` to the actual addon Url.